scripts/e2e/values-e2e.yaml

```diff
@@ -15,6 +15,14 @@ config:
     EncryptionKey: password
   StandardTransformerConfig:
     ImageName: ghcr.io/caraml-dev/merlin-transformer:0.31.3
+  BatchConfig:
```

Comment:
just to confirm this is already supported by the current [chart](https://github.com/caraml-dev/helm-charts/blob/main/charts/merlin/templates/merlin-config-secret.yaml#L10)

Reply:
Yep, the chart just dumps everything in this `config` field (which is where the `BatchConfig` will be) into the merlin configs. So technically the chart doesn't need to be changed in any way (we might just want to add the defaults in that's all)